### PR TITLE
Fixes #14311: Add bundle dedicated to the ncf tests

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14311

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.success.cf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      "unknown_service_name"  string => "unknown";
+
+  classes:
+    redhat|suse::
+      "enabled" expression => returnszero("chkconfig ${service_name} on", "useshell");
+    debian::
+      "enabled" expression => returnszero("update-rc.d ${service_name} enable", "useshell");
+
+  methods:
+    "enable"  usebundle => set_dry_run_mode("true");
```
Comment (Fdall) [inline tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.success.cf]: Non-sense

**Comment by Fdall** [comment]
Commit modified

File: tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.enforce.success.cf [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      "unknown_service_name"  string => "unknown";
+
+  classes:
+    redhat|suse::
+      "enabled" expression => returnszero("chkconfig ${service_name} on", "useshell");
+    debian::
+      "enabled" expression => returnszero("update-rc.d ${service_name} enable", "useshell");
+  methods:
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "success", "1");
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "success", "2");
```
Comment (ncharles) [inline tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.enforce.success.cf]: why should it be a success ?

Comment (ncharles) [reply tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.enforce.success.cf]: actually, the second one should be unknown_service_name

File: tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.enforce.error.cf [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+  classes:
+    redhat|suse::
+      "enabled" expression => returnszero("chkconfig ${service_name} off", "useshell");
+    debian::
+      "enabled" expression => returnszero("update-rc.d ${service_name} disable", "useshell");
+
+  methods:
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "error", "1");
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "error", "2");
```
Comment (ncharles) [inline tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.enforce.error.cf]: you are not correctly defining the second class (for unknown_service_name)

File: tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.success.cf [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  classes:
+    redhat|suse::
+      "enabled" expression => returnszero("chkconfig ${service_name} on", "useshell");
+    debian::
+      "enabled" expression => returnszero("update-rc.d ${service_name} enable", "useshell");
+
+  methods:
+    "any"     usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "success", "1");
+    "any"     usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "success", "2");
```
Comment (ncharles) [inline tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.success.cf]: here also: unknown_service_name

File: tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.error.cf [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+  classes:
+    redhat|suse::
+      "enabled" expression => returnszero("chkconfig ${service_name} off", "useshell");
+    debian::
+      "enabled" expression => returnszero("update-rc.d ${service_name} disable", "useshell");
+
+  methods:
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "error", "1");
+    "any" usebundle => define_expected_classes("service_check_started_at_boot_${service_name}", "error", "2");
```
Comment (ncharles) [inline tests/acceptance/30_generic_methods/unsafe/service_check_started_at_boot.audit.error.cf]: again unknown_service_name

**Comment by fanf** [comment]
ping @Fdall

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
PR rebased

**Pull request merged**